### PR TITLE
Add v1.14.1. [release]

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

**Comment by roidelapluie** [comment]
:+1:

File: 1.14/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-FROM cimg/base:2020.01
+FROM cimg/base:2020.03
 
 LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
 
-ENV GO_VERSION=1.14
-ENV GO_SHA=
+ENV GO_VERSION=1.14.1
+ENV GO_SHA=fake-sha
```
Comment (KyleTryon) [inline 1.14/Dockerfile]: What is the purpose of this?

Comment (felicianotech) [reply 1.14/Dockerfile]: The first line sets the Go version and the second line is a placeholder for when SHA support is built.

**Comment by felicianotech** [comment]
@KyleTryon Can I get an approval review here so I can get this published?

**Comment by roopakv** [comment]
i could be wrong, but the released image still has `1.14` not `1.14.1`

**Pull request merged**